id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/service/impl/CredentialRequestServiceImpl.java

```diff
@@ -114,6 +118,7 @@ public ResponseWrapper<CredentialIssueResponse> createCredentialIssuance(Credent
 			credential.setCreatedBy(IdRepoSecurityManager.getUser());
 			credential.setStatusComment("Request created");
 			credentialDao.save(credential);
+			cacheUtil. setCredentialTransaction(requestId, credential);

```

Comment:
Is it required to cache the credentialEntity?

Reply:
I updated the code and now caching response dto of get status endpoint instead of credentialEntity.